src/variables/body/variablesSearch.tsx

```diff
@@ -0,0 +1,126 @@
+// Copyright (c) Jupyter Development Team.
```

Comment:
I sould suggest renaming this file to `search.tsx` and renaming the class to `class Search extends Widget`.